katsdpbfingest/bf_ingest_server.py

```diff
@@ -403,17 +404,28 @@ def __init__(self, args: argparse.Namespace, loop: asyncio.AbstractEventLoop) ->
 
     def update_counters(self, counters: ReceiverCounters) -> None:
         timestamp = time.time()
-        counter_sensors = ['bytes', 'packets', 'batches',
-                           'heaps', 'too-old-heaps', 'incomplete-heaps', 'metadata-heaps',
-                           'bad-timestamp-heaps', 'bad-channel-heaps', 'bad-length-heaps']
-        gauge_sensors = ['max-batch']
-        for name in counter_sensors + gauge_sensors:
-            sensor_name = 'input-{}{}'.format(name, '' if name in gauge_sensors else '-total')
+        # Map sensor name to counter name
+        sensors = {
+            'input-bytes-total': 'katsdpbfingest.bytes',
+            'input-packets-total': 'packets',
+            'input-batches-total': 'batches',
+            'input-heaps-total': 'katsdpbfingest.data_heaps',
```

Comment:
Why are some counter names prefixed with `katsdpbfingest` and others not?

Reply:
The ones that don't have it are provided by spead2 itself. The ones with the `katsdpbfingest.` prefix are computed by katsdpbfingest - I prefix them to avoid any potential conflicts with future spead2-provided counters.

Reply:
It may have been better to call it `input-data-heaps-total` or something in the past (the `-total` is a Prometheus convention for monotonic counters), but I don't particularly feel like changing all the services that use `input-heaps-total` to mean data heaps and all the monitoring that consumes that name.